src/operator/src/expr_factory.rs

```diff
@@ -483,6 +483,15 @@ pub(crate) fn to_alter_expr(
         AlterTableOperation::RenameTable { new_table_name } => Kind::RenameTable(RenameTable {
             new_table_name: new_table_name.to_string(),
         }),
+        AlterTableOperation::AlterColumnFulltext {
+            column_name,
+            options,
+        } => Kind::ChangeFulltext(ChangeFulltext {
+            column_name: column_name.value.to_string(),
+            enable: options.enable,
+            analyzer: options.analyzer,
+            case_sensitive: options.case_sensitive,
+        }),
```

Comment:
```suggestion
        AlterTableOperation::AlterColumnFulltext {
            column_name,
            options,
        } => Kind::ChangeFulltext(ChangeFulltext {
            column_name: column_name.value.to_string(),
            enable: options.enable,
            analyzer: match options.analyzer {
                Some(FulltextAnalyzer::Chinese) => Some(Analyzer::Chiniese as i32),
                Some(FulltextAnalyzer::English) => Some(Analyzer::English as i32),
                None => None,
            },
            case_sensitive: options.case_sensitive,
        }),
```

And the found a proto typo: `Chiniese`